Packs/Lumu/Integrations/Lumu/Lumu.py

```diff
@@ -0,0 +1,1092 @@
+    page = args.get("page")
+    items = args.get("items")
+    adversary_types = args.get("adversary_types")
+    labels = args.get("labels")
+
+    adversary_types = argToList(adversary_types)
+    if labels and isinstance(labels, str):
+        labels = [int(label) for label in labels.split(",")]
+
+    response = client.retrieve_open_incidents_request(
+        page, items, adversary_types, labels
+    )
+    command_results = CommandResults(
+        outputs_prefix="Lumu.RetrieveOpenIncidents",
+        outputs_key_field="id",
+        outputs=response,
+        raw_response=response,
+        readable_output=tableToMarkdown('Incidents', response.get('items', []), headerTransform=pascalToSpace, removeNull=True)
+    )
+
+    return command_results
+
+
+def retrieve_muted_incidents_command(
+    client: Client, args: Dict[str, Any]
+) -> CommandResults:
+    page = args.get("page")
+    items = args.get("items")
+    adversary_types = args.get("adversary_types")
+    labels = args.get("labels")
+
+    adversary_types = argToList(adversary_types)
+    if labels and isinstance(labels, str):
+        labels = [int(label) for label in labels.split(",")]
+
+    response = client.retrieve_muted_incidents_request(
+        page, items, adversary_types, labels
+    )
+    command_results = CommandResults(
+        outputs_prefix="Lumu.RetrieveMutedIncidents",
+        outputs_key_field="id",
+        outputs=response,
+        raw_response=response,
+        readable_output=tableToMarkdown('Incidents', response.get('items', []), headerTransform=pascalToSpace, removeNull=True)
+    )
+
+    return command_results
+
+
+def retrieve_closed_incidents_command(
+    client: Client, args: Dict[str, Any]
+) -> CommandResults:
+    page = args.get("page")
+    items = args.get("items")
+    adversary_types = args.get("adversary_types")
+    labels = args.get("labels")
+
+    adversary_types = argToList(adversary_types)
+    if labels and isinstance(labels, str):
+        labels = [int(label) for label in labels.split(",")]
+
+    response = client.retrieve_closed_incidents_request(
+        page, items, adversary_types, labels
+    )
+    command_results = CommandResults(
+        outputs_prefix="Lumu.RetrieveClosedIncidents",
+        outputs_key_field="id",
+        outputs=response,
+        raw_response=response,
+        readable_output=tableToMarkdown('Incidents', response.get('items', []), headerTransform=pascalToSpace, removeNull=True)
+    )
+
+    return command_results
+
+
+def retrieve_endpoints_by_incident_command(
+    client: Client, args: Dict[str, Any]
+) -> CommandResults:
+    lumu_incident_id = args.get("lumu_incident_id")
+    page = args.get("page")
+    items = args.get("items")
+
+    response = client.retrieve_endpoints_by_incident_request(
+        lumu_incident_id, page, items
+    )
+    command_results = CommandResults(
+        outputs_prefix="Lumu.RetrieveEndpointsByIncident",
+        outputs_key_field="id",
```

Comment:
in this case its not `id`
```suggestion
        outputs_key_field="label",
```

Reply:
done in this f47ed2c15d6f2c181b37fe28e7a84706396c6845